lua/ttt2/libraries/migrations.lua

```diff
@@ -0,0 +1,122 @@
+---
+-- A TTT2 version migrations library
+-- @author ZenBre4ker
+-- @module migrations
+
+if SERVER then
+	AddCSLuaFile()
+end
+
+migrations = {}
+migrations.folderPath = "terrortown/migrations/"
+migrations.baseFolderPath = migrations.folderPath .. "base/"
+migrations.databaseName = "ttt2_migrations"
+migrations.savingKeys = {
+	executedAt = {
+		typ = "string",
+		default = ""
+	},
+	statusMessage = {
+		typ = "string",
+		default = "Success"
+	},
```

Comment:
This is somewhat redundant, `executedAt` should only be set after the migration was fully executed and therefore implies success

Reply:
executedAt is just the time, while statusMessage could contain an error message in case of failure. Could make it easier to see when migrations failed and why for bug reports.

Reply:
But maybe an actual log is better.